pkg/init/cmd/service/runc.go

```diff
@@ -103,9 +151,26 @@ func runcInit(rootPath string) int {
 
 		cleanup(path)
 		_ = os.Remove(pidfile)
+
+		// dump the log file outputs to os.Stdout/os.Stderr
+		err = dumpFile(os.Stdout, stdoutFile)
+		if err != nil {
```

Comment:
You can use the `if err := .... ; err != nil {` style here (and below)

Reply:
Yeah, good point there too. Done.